lib/charms/opensearch/v0/opensearch_plugin_manager.py

```diff
@@ -277,14 +251,16 @@ def _compute_settings(
         self, config: OpenSearchPluginConfig
     ) -> Tuple[Dict[str, str], Dict[str, str]]:
         """Returns the current and the new configuration."""
+        if not self._charm.opensearch.is_node_up() or not self.is_ready_for_api():
```

Comment:
Yes, I think this particular change predates our fix for the API calls where we iterate over the multiple addresses, instead of always sending the same API request to the same unit on every retry... I think it is safe to just have the is_ready_for_api().